### ISSUE TITLE
Barrier / gate is not rendered

### ISSUE THREAD

**Issue by brunetton** [organicmaps/organicmaps#1197]
Hi, I'm using Organic Maps for a while and I'm SOOO HAPYYY to have found a real open source fork of MapsWithMe 🤩

But yesterday I came across something missing on the map: a barrier that is not rendered:

### not rendered

https://www.openstreetmap.org/node/288354527

```
barrier=swing_gate
bicycle=yes
foot=yes
horse=yes
motor_vehicle=designated
```

![IMG_20210827_165255](https://user-images.githubusercontent.com/86322/131148427-5df116e2-6ba9-4ab7-8d87-125e91cc793c.jpg)

(but rendered on OSM standard): 
![image](https://user-images.githubusercontent.com/86322/131142883-ac0a2f33-e09c-496a-8c62-1549d5db3a85.png)

### but this one is correctly rendered

https://www.openstreetmap.org/node/8327476375

```
barrier=lift_gate
bicycle=yes
foot=yes
horse=yes
```

**Comment by Deinrad** [comment]
Hi!
Today I learned that barrier=gate has only a very small dot in OM.
I think this is very easily overlooked. It might be better to replace the icon with something bigger.
This specific case applies to a barrier that has no further specifications concerning who/what can pass. I guess this is often the case. In my specific case the gate was locked so nobody could pass.

The case:
https://www.openstreetmap.org/node/3419406372

![BC166038-77E1-4D9D-8BE3-7161EDC5A213](https://user-images.githubusercontent.com/94761015/142738777-ff4746d0-e7be-47d2-8289-ac465658d3d5.png)

**Comment by pastk** [comment]
> [The wiki](https://wiki.openstreetmap.org/wiki/Tag:barrier%3Dgate) doesn't precise this as far as I see. Do you know for sure the exact meanings of those tags combinations ?

Having read the wiki I can second that the whole `access` thing is quite complicated, `access=yes` could be overriden by `vehicle=` or `motorcar=` etc. My overall impression is it requires the mapper be very knowledgeable and accurate. And this probably mean there are a lot of tagging mistakes.

Taking it into consideration I think its important to render _ANY_ barrier on the map regardless of its "access" tags. If there is a physical structure - then show its there. If the barrier is rendered doesn't mean its closed (or open), but at least it allows the user to prepare in advance - e.g. if in doubt then ask local people, call a friend who've been there etc. before going there by car.

As for the routing I think the gate should be considered passable e.g. by car only if no of `access=`, `vehicle=`, `motor_vehicle=`, `motorcar=` etc. tags contradicts it.
Another good options is to display gates with uncertain status on the route with a special icon similar to e.g. "unpaved road" or "ferry crossing" icon.

**Comment by rmtsrc** [comment]
I also recently had this issue while hiking. I was walking along a coastal footpath shown in Organic Maps, but came across a big metal barrier with a no entry sign preventing access and noted that I should add it to Open Street Map later, as there was nowhere else to go (being along side a cliff face and next to the sea) I had to walk back and find another footpath.

However, when I looked on Open Street Map a gate had already been placed there and was tagged with `access: no`, `barrier: gate` 4 years ago. It also showed the open part of the footpath (which I was on) as a red dashed line, while the footpath on the other side of the barrier was marked with a gray dashed line with `access: no` on the footpaths tags:

![Screenshot from 2022-09-17 16-30-49](https://user-images.githubusercontent.com/144435/190865135-6e982ee7-a7ed-458e-ba74-d615b7175089.png)

Looking back in Organic Maps it shows the gate as a tiny dot (which I didn't spot while hiking) and the footpath is marked as open on both sides of the barrier:

<img src="https://user-images.githubusercontent.com/144435/190865373-6b8f19e9-0df6-4e03-9931-7d5f579d8a77.png" alt="Screenshot_20220916-104807_Organic Maps" width="360">

I think it would be a good idea to show gates on in Organic Maps with the same or similar icon to Open Street Map and also indicate inaccessible footpaths, to help notify future hikers of inaccessible footpaths.
